packages/@guidesmiths/cuckoojs-schematics/src/pr-template/files/azuredevops/.azuredevops/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+### Description (What I did in this PR?)
+
+- (purpose of the PR, how did you implement it)
+
+### Evidences (visual proof of the changes implemented)
+
+- (some screenshoots or log traces with the effects)
+
+### Review process (How to review this PR?)
+
+- (explain how to review it)
+
+### Link to board's task
+
+- (paste a link to the correspondent task or subtask)
+
+### Pre-approval checklist (Before submitting this PR, I made sure...)
+
+- [x] The code builds clean without any errors or warnings
+- [x] I've added/updated tests
+- [x] I've added/updated documentation if needed
+- [x] I've managed how this PR will be reviewed
```

Comment:
Instead of duplicating the same PR twice, we can make use of schematics features to create the folder that contains the file dynamically (https://www.kevinschuchard.com/blog/2019-05-17-iterpolating-schematics/)

Perhaps something similar to this works:

```
files
|-.__options.gitProvider__
  |- pull_request_template.md
```

Reply:
it looks great. I'm a bit concerned if we add another provider in the future, but let's think what we have now and planned in the short/medium term 👍

Reply:
done